src/me/drton/jmavsim/Simulator.java

```diff
@@ -103,15 +108,20 @@ public Simulator() throws IOException, InterruptedException, ParserConfiguration
 
         // Create environment
         SimpleEnvironment simpleEnvironment = new SimpleEnvironment(world);
-        Vector3d magField = new Vector3d(0.2f, 0.0f, 0.5f);
-        Matrix3d magDecl = new Matrix3d();
-        magDecl.rotZ(11.0 / 180.0 * Math.PI);
-        magDecl.transform(magField);
-        simpleEnvironment.setMagField(magField);
+        Vector3d magField = new Vector3d(0.21523, 0.0f, 0.42741);
```

Comment:
Why are these magic numbers double in the code?

Also, please add a comment where they come from.